Fix import problem introduced with previous pull request

swistakm opened the pull request:
Was unable to install with setup.py

swistakm committed:
Fix imports

Was unable to install with setup.py

swistakm committed:
Tasks refactorization

Separates tasks from the way they can be actually
executed

swistakm committed:
New management commands

Allows installing tasks using different backends. Possible
backends at the moment:
- kronos
- celery

Methods added:
- installsensors
- uninstallsensors

Backend is determined by the variable
SKWISSH_TASK_BACKEND in settings.py. Default value
 is 'kronos'.

swistakm committed:
Information in README about how to change task backend

swistakm committed:
installtasks changed to installsensors in README

Changed because new method can be used for both
backends and user should now choose this one. If not
 so he can make misteke if switches to celery.

swistakm committed:
Fix README

swistakm committed:
merge

rsaikali commented:
I'm unable to run setup.py with your commit.
Are you sure everything is OK with your files ?

Can't merge in current state...

swistakm commented:
I will check this...

swistakm commented:
I fixed everything few days ago. I added some changes after first commit which were not intended in this pull request and introduced mentioned problem with imports. It's why I closed this request and made #17. New pull request describes better all changes I did.
